security/pkg/k8s/chiron/controller.go

```diff
@@ -60,7 +60,7 @@ const (
 	certReadInterval = 500 * time.Millisecond
 )
 
-var certWatchTimeout = 5 * time.Second
+var certWatchTimeout = time.Hour
```

Comment:
Why does the timeout need to be increased from 5 second to 1 hour?

Reply:
1hr seems a bit excessive. Even taking 5s (I think it ends up being 10s somehow) is likely a sign the CA is having issues

Reply:
trying to match kubelet timeout. Since the DNS cert is essentially a pre-requisite (assuming CA is enabled) for Istiod to start, is there a problem with increasing this timeout value?

Reply:
Refactored this to make the cert watch Timeout = 1 minute for both workload certificates and DNS certificates. PTAL